docs/rule/invocable-blacklist.md

```diff
@@ -26,4 +26,9 @@ This rule **forbids** the following:
 
 ## Configuration
 
+One of these:
+
 * array of strings - helpers or components to blacklist (using kebab-case names like `nested-scope/component-name`)
+* array of objects with the following keys:
+  * `name` - helpers or components to blacklist (using kebab-case names like `nested-scope/component-name`)
```

Comment:
Should we allow an array too (so that you could share a single message for a list of names)? I’m thinking that might seem similar to the overrides stuff :thinking:

Also, seems fine to punt for now.

Reply:
Good catch, that was actually my original intention, so I have updated each object to take an array of names.